src/GoodsList.js

```diff
@@ -0,0 +1,90 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import './App.css';
+
+export class GoodsList extends React.Component {
+  state = {
+    sortBy: '',
+    reversed: false,
```

Comment:
you don't need to store this values in the state, on click on some button just do needed changes with goods array